cache/caffeine-cache/src/it/settings.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~   Licensed under the Apache License, Version 2.0 (the "License");
+~   you may not use this file except in compliance with the License.
+~   You may obtain a copy of the License at
+~
+~   http://www.apache.org/licenses/LICENSE-2.0
+~
+~   Unless required by applicable law or agreed to in writing, software
+~   distributed under the License is distributed on an "AS IS" BASIS,
+~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~   See the License for the specific language governing permissions and
+~   limitations under the License.
+-->
+
+<settings>
+    <mirrors>
+        <mirror>
```

Comment:
I am not sure I understand all this new integration testing stuff. It seems like it is a lot of XML to maintain, and complexity to the build (the need to install artifacts, for example). Are you adding it to catch some class of error that cannot be caught by "normal" surefire tests?

Reply:
it actually is the "old" style of integration tests. These are small maven projects. The idea is to run all the core tests with the specific cache layer enabled. So the "normal" core tests run with the default cache configuration and all the tests here run with the caffeine/guava/no-op cache.